packages/package_image_1_4_1/tool_dependencies.xml

```diff
@@ -0,0 +1,17 @@
+<tool_dependency>
+    <package name="Image" version="1.4.1">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">https://codeload.github.com/francescortiz/image/tar.gz/master</action>
+                <action type="make_directory">$INSTALL_DIR/lib/python</action>
+                <action type="shell_command">export PYTHONPATH=$PYTHONPATH:$INSTALL_DIR/lib/python &amp;&amp; python setup.py install --home $INSTALL_DIR --install-scripts $INSTALL_DIR/bin</action>
+                <action type="set_environment">
+                    <environment_variable name="PYTHONPATH" action="append_to">$INSTALL_DIR/lib/python</environment_variable>
+                    <environment_variable name="PYTHONPATH_IMAGE" action="set_to">$INSTALL_DIR/lib/python</environment_variable>
+                    <environment_variable name="PATH" action="prepend_to">$INSTALL_DIR/bin</environment_variable>
```

Comment:
@mvdbeek no you do not just do a PR because NO that's not easier *for* @fabiorjvieira and myself. Please ask the @fabiorjvieira 's question.